fili-core/src/main/java/com/yahoo/bard/webservice/web/util/QuerySignedCacheService.java

```diff
@@ -168,12 +213,15 @@ public void writeCache(
                 //mark and log the cache put failure
                 CACHE_SET_FAILURES.mark(1);
                 BardQueryInfo.getBardQueryInfo().incrementCountCacheSetFailures();
-                BardQueryInfo.getBardQueryInfo().addPutFailureInfo(
+                BardQueryInfo.getBardQueryInfo().addCachePutInfo(
```

Comment:
This is now typically happening after query completion so the BardQueryInfo isn't useful for logging.  (and may already have been cleared)